StripeConnect/StripeConnect/Source/Internal/Webview/ConnectComponentWebView.swift

```diff
@@ -25,6 +26,16 @@ class ConnectComponentWebView: ConnectWebView {
     /// The current notification center instance
     private let notificationCenter: NotificationCenter
     
+    private let setterMessageHandler: OnSetterFunctionCalledMessageHandler = .init()
+    
+    let activityIndicator: ActivityIndicator = {
+        let activityIndicator = ActivityIndicator()
+        activityIndicator.hidesWhenStopped = true
+        activityIndicator.tintColor = .gray
```

Comment:
Open question around what color to use here. Determining if this spinner on web uses appearance or not.

Reply:
If you do end up addressing these changes in this PR, then please link the associated ticket in the PR description:
https://jira.corp.stripe.com/browse/MXMOBILE-2743

Reply:
Code refs for web logic:
- frontend/embedded-theming/src/getAppearanceValuesWithDefaults.ts#L120
- frontend/embedded-theming/src/tokenOverrides.ts#L191-194
- frontend/embedded-theming/src/contrastUtils.ts#L329-405